docs/en/3-Pipelines/.spelling

```diff
@@ -0,0 +1,3 @@
+KFP-specific
```

Comment:
Do we have a good alternative? Like "pipeline-specific"? If not, probably just toss it in the English root.